tests/test_methods_nai.py

```diff
@@ -0,0 +1,71 @@
+"""This test aims to compare to previous PDA and PDAW results that were published and compared to exact QD.
+We use NaI as an example with two different laser pulses. If this test fails, the results produced by the code are unreliable!"""
+
+from pathlib import Path
+
+import numpy as np
+
+from promdens.promdens import InitialConditions, LaserPulse
+
+# path to the tests folder
+path = Path(__file__).parent.absolute()
+
+def test_pdaw_nai(tmp_path):
+    ics = InitialConditions(nsamples=1000, nstates=1, input_type='file')
+
+    ics.read_input_data(fname=path/'NaI_reference/test_data_nai.dat', energy_unit='a.u.', tdm_unit='a.u.')
+
+    fstoau = 41.341374575751
+    pulse = LaserPulse(omega=0.13520905, fwhm=20*fstoau, envelope_type='gauss', lchirp=0, t0=0)
+
+    ics.calc_field(pulse=pulse)
+
+    ics.windowing()
+
+    # moving output to a temporary path dedicated to this test
+    tmp_folder = tmp_path / "pdaw"
+    tmp_folder.mkdir()
+    Path('pdaw.dat').rename(tmp_folder / 'pdaw.dat')
```

Comment:
the Path().rename() method unfortunately does not work if the original and target paths are not on the same filesystem. On my laptop I am getting this error

```
OSError: [Errno 18] Invalid cross-device link: 
'pdaw.dat' -> '/tmp/pytest-of-hollas/pytest-8/test_pdaw_nai0/pdaw/pdaw.dat'
```

Instead I think the file shoud be created in `tmp_path` in the first place. I'd add an extra argument to `windowing` function, perhaps names `output_fname` or something like that so you can pass the path when calling the function. And same for `sample_initial_conditions`. The arguments can have default values so you don't have to change the code in the main script and only use it here in tests. WDYT?

```
    def windowing(self, output_fname='pdaw.dat'):
```